setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import setup
+
+setup(
+    name='libero schemas',
+    version='0.1',
+    description='libero xml api schemas',
```

Comment:
```suggestion
    description='Libero data model schemas',
```

This will contain non HTTP-schemas in the future too (ie event bus).